sklearn/ensemble/_weight_boosting.py

```diff
@@ -1166,3 +1171,8 @@ def staged_predict(self, X):
 
         for i, _ in enumerate(self.estimators_, 1):
             yield self._get_median_predict(X, limit=i)
+
+    def __sklearn_tags__(self):
```

Comment:
Same here, shall we have a single shared definition in `BaseWeightBoosting`?

Reply:
done in 705c414